### PR TITLE
Use correct icon tint in voice message preview player

### PR DESCRIPTION
**Pull request by julioromano**
Action item from design review.

### CODE REVIEW

**Comment by julioromano** [comment]
Screenshots: https://github.com/vector-im/element-x-android/actions/runs/6669054238

**Review by jonnyandrew — APPROVED** [review]

File: tests/uitests/src/test/snapshots/images/ui_S_t[l.textcomposer.components_null_VoiceMessagePreview-D-16_16_null,NEXUS_5,1.0,en].png
Comment (jmartinesp) [inline tests/uitests/src/test/snapshots/images/ui_S_t[l.textcomposer.components_null_VoiceMessagePreview-D-16_16_null,NEXUS_5,1.0,en].png:1]: This is quite weird, now it looks as if it were always enabled.

File: tests/uitests/src/test/snapshots/images/ui_S_t[l.textcomposer.components_null_VoiceMessagePreview-N-16_17_null,NEXUS_5,1.0,en].png
Comment (jmartinesp) [inline tests/uitests/src/test/snapshots/images/ui_S_t[l.textcomposer.components_null_VoiceMessagePreview-N-16_17_null,NEXUS_5,1.0,en].png:1]: And here every button looks disabled.

File: tests/uitests/src/test/snapshots/images/ui_S_t[l.textcomposer.components_null_VoiceMessagePreview-D-16_16_null,NEXUS_5,1.0,en].png
Comment (julioromano) [reply tests/uitests/src/test/snapshots/images/ui_S_t[l.textcomposer.components_null_VoiceMessagePreview-D-16_16_null,NEXUS_5,1.0,en].png:1]: I agree @callumu  what d'you think?

**Pull request merged**